RoddyCore/src/de/dkfz/roddy/config/ToolEntry.java

```diff
@@ -217,6 +219,39 @@ public ToolStringParameter clone() {
         }
     }
 
+    public static class ToolFileParameterCondition {
+
+        private String condition;
+
+        private Boolean pureBoolean;
+
+        public ToolFileParameterCondition(boolean pureBoolean) {
+            this.pureBoolean = Boolean.valueOf(pureBoolean);
+        }
+
+        public ToolFileParameterCondition(String condition) {
+            if(condition.startsWith("conditional:"))
+                this.condition = condition.substring(12);
+            else {
+                this.pureBoolean = RoddyConversionHelperMethods.toBoolean(condition, true);
```

Comment:
I think, the default "true" is not backwards compatible. Usually, empty values with Boolean were either false, or the default value obtained with toBoolean, which depended on the call site. With this, all unspecified values will be true. What we had as "Boolean" before was actually a tri-state: true, false, $defaultValue